common/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/version/SimpleRange.java

```diff
@@ -48,8 +48,8 @@
     private final VersionModifier modifier;
 
     private SimpleRange(VersionId versionId, VersionModifier modifier) {
-        this.versionId = versionId;
-        this.modifier = modifier;
+        this.versionId = Assert.requireNonNull(versionId, "versionId");
```

Comment:
private method only called from within this class and calling code never passes null
do we still need the checks?